SpaceGame/Resources/Modules/Game.js

```diff
@@ -89,6 +92,61 @@ Game.prototype.createScene2D = function() {
     this.camera = camera;
     this.viewport = viewport;
 
+    if ( Atomic.input.numJoysticks > 0 ) {
+
+         this.jsid = 0
+
+        scene.subscribeToEvent("JoystickConnected", function(ev) {
+            this.jsid = ev.JoystickID; // get the joystick id for future calls.
+        });
```

Comment:
Hey, cool on demand joystick?  If so, already nicer than 90% of the games I have played where you have to restart if you forgot to turn gamepad on :)